Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -417,6 +419,7 @@ public AndConstraint<TAssertions> BeEmpty(string because = "", params object[] b
     public AndConstraint<SubsequentOrderingAssertions<T>> BeInAscendingOrder(
         IComparer<T> comparer, string because = "", params object[] becauseArgs)
     {
+        Guard.ThrowIfArgumentIsNull(comparer, nameof(comparer), "Cannot assert collection ordering without specifying a comparer.");
```

Comment:
The custom message doesn't seem to be adding much value here from my perspective (as a consumer). I'd suggest removing in cases like this to reduce noise in the codebase.